src/hb-subset-plan.cc

```diff
@@ -596,3 +602,108 @@ hb_subset_plan_destroy (hb_subset_plan_t *plan)
 
   hb_free (plan);
 }
+
+/**
+ * hb_subset_plan_old_to_new_glyph_mapping:
+ * @plan: a subsetting plan.
+ *
+ * Return value:
+ * the mapping between glyphs in the original font to glyphs
+ * in the subset that will be produced by @plan.
+ *
+ * Since: REPLACEME
+ **/
+const hb_map_t*
+hb_subset_plan_old_to_new_glyph_mapping (const hb_subset_plan_t *plan)
+{
+  return plan->glyph_map;
+}
+
+/**
+ * hb_subset_plan_old_to_new_glyph_mapping:
+ * @plan: a subsetting plan.
+ *
+ * Returns the mapping between glyphs in the subset that will be produced by
+ * @plan and the glyph in the original font.
+ *
+ * Since: REPLACEME
+ **/
+const hb_map_t*
+hb_subset_plan_new_to_old_glyph_mapping (const hb_subset_plan_t *plan)
+{
+  return plan->reverse_glyph_map;
+}
+
+/**
+ * hb_subset_plan_old_to_new_glyph_mapping:
+ * @plan: a subsetting plan.
+ *
+ * Returns the mapping between codepoints in the original font and the
+ * associated glyph id in the original font.
+ *
+ * Since: REPLACEME
+ **/
+const hb_map_t*
+hb_subset_plan_codepoint_to_old_glyph_mapping (const hb_subset_plan_t *plan)
```

Comment:
Should we even return this? This is available from the original face readily, right?

Reply:
In the case where the subset was specified using only glyph id's this provides a method to determine which codepoints are going to be in the final subset.

Reply:
I see. Sure.